gspread/models.py

```diff
@@ -269,6 +271,9 @@ def add_worksheet(self, title, rows, cols):
             }]
         }
 
+        if index:
```

Comment:
```suggestion
        if index or index == 0:
```

If `index` is set to 0, this will return false and not send the index to google. Setting an `index` of 0 is very useful when you want your current worksheet to be the first worksheet within the spreadsheet.